atbash_cipher/atbash_cipher_task.py

```diff
@@ -0,0 +1,27 @@
+# Task from https://exercism.org/tracks/python/exercises/atbash-cipher
+
+def encode(plain_text):
+    plain_str = "abcdefghijklm0123456789nopqrstuvwxyz"
+    cipher_str = "zyxwvutsrqpon0123456789mlkjihgfedcba"
+    result = ""
+    letters_counter = 0
+    for symbol in plain_text.lower():
+        if letters_counter == 5 and symbol in plain_str:
+            result += " "
+            letters_counter = 0
+        if symbol in plain_str:
```

Comment:
Please change these if statement as it has the same condition as in previous one